assignment1.txt

```diff
@@ -0,0 +1,10 @@
+1.  A compiler turns the coding which is readable for humans into a binary form, so that a computer can read it.
+
+2. Compiler errors are when they occur while the code is being compiled, whereas a runtime error occurs while the computor is running the code.
```

Comment:
right - compile time errors have more to do with syntax, whereas runtime have more to do with logic.